boards/arm/thingy52_nrf52832/thingy52_nrf52832.dts

```diff
@@ -35,14 +35,17 @@
 		led0: led_0 {
 			gpios = <&sx1509b 5 GPIO_ACTIVE_LOW>;
 			label = "Green LED";
+			//vin-supply = <&vdd_pwr>;
```

Comment:
Why are these commented out?

Reply:
Because Zephyr's devicetree binding language doesn't allow inclusion of `power.yaml` to extend the [child-binding](https://github.com/zephyrproject-rtos/zephyr/blob/762b806c3d4bf2f1004052558141913f33462cfa/dts/bindings/gpio/gpio-leds.yaml#L8).  That will have to be fixed, but only when this starts to be usable which requires device dependency.

Reply:
ok